seed/models/scenarios.py

```diff
@@ -91,11 +91,17 @@ def copy_initial_meters(self, source_scenario_id):
         from seed.models.meters import Meter
 
         source_scenario = Scenario.objects.get(pk=source_scenario_id)
+        try:
+            property_ = PropertyView.objects.get(state=self.property_state).property
+        except PropertyView.DoesNotExist:
+            property_ = None
 
         for source_meter in source_scenario.meter_set.all():
             # create new meter and copy over the readings from the source_meter
             meter = Meter.objects.get(pk=source_meter.id)
             meter.pk = None
             meter.scenario_id = self.id
+            if property_ is not None:
+                meter.property = property_
```

Comment:
I think having a meter without a property association makes sense in the bigger picture, but as it currently stands, I've never seen a meter without a property associated to it. Maybe this is something we chat about in person.

One thought: If we don't have a related property, would the newly created meter be visible on the frontend?

Reply:
Per conversation, we'll removing this conditional